docs/backplane/backplane_depop.md

```diff
@@ -0,0 +1,57 @@
+## `JD` depopulations
+
+For both _true_ and _mirror_ type, the depopulations are:
+
+- F -> P: `JD2,3` depopulate
+- P -> D: `JD10,11` depopulate
+
+
+## `JP` depopulations
+
+### True backplanes
+For F -> P:
+
+|          | P1W   | P1E   | P2W   | P2E   | P3   | P4   |
+|----------|-------|-------|-------|-------|------|------|
+| JP0      | X     | O     |       | X     | X    | X    |
+| JP1      | X     | O     | O     | X     | X    | O    |
+| JP2      | X     | O     | O     | X     | X    | O    |
+| JP3      | X     | O     |       | X     | X    | X    |
+| JP4      | X     | O     |       | X     | X    | X    |
+| JP5      | X     | O     |       | X     | X    |      |
+| JP6      | X     | O     |       | X     | X    |      |
+| JP7      | X     | O     |       | X     | X    | X    |
+| JP8      | X     |       |       | X     | X    | X    |
+| JP9      | X     |       |       | X     | X    |      |
+| JP10     | X     |       |       | X     | X    |      |
+| JP11     | X     |       |       | X     | X    | X    |
+
+!!! note
+    - **X**: Hybrid is present on both F and P (but may depopulate a few elinks)
+    - **O**: Hybrid is depopulated on P
+    - blank: not present
+
+!!! info
+    P1-P3 are on P2B2, P4 are on Inner BB.
```

Comment:
Not correct. I agree it's very helpful to identify which are on P2B2 and which are on Inner BB, but please see backplane schematics for JPU/JPL for the correct info.

Reply:
I've looked at the true BP schematic, and I think the main fix is: For `JP8-11`, both P3 and P4 are on the `JPL` connectors.